### ISSUE TITLE
Controller Time Text Incorrect

### ISSUE THREAD

**Issue by DanielRobert1** [m3uzz/date_time_picker#55]
This happens when 24hr mode is disabled, text returns in 12hr Mode which cannot be parsed

**Comment by Haseebarshad849** [comment]
@DanielRobert1 did you find the solution?

**Comment by DanielRobert1** [comment]
No, I had to downgrade to a lower version, I might fork the repo and fix it

**Comment by DanielRobert1** [comment]
downgrade to 2.0.0 that fixed the error,
